pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+name = "alfalfa"
+version = "0.1.0"
+description = ""
+authors = ["Nicholas Long <nicholas.long@nrel.gov>"]
+
+[tool.poetry.dependencies]
+python = "^3.6.1"
+importlib-metadata = "3.4.0"
+boto3 = "1.17.8"
+mlep = "0.1.1.dev3"
+pymongo = "3.11.3"
+redis = "3.5.3"
+requests = "2.25.1"
+symfit = "0.5.3"
+# These libraries are used for the data generator/testcase. They are pinned to older versions
+# as the min python version is 3.7
+pandas = "0.24.2"
+scipy = "1.5.4"
+numpy = "1.19.5"
+
+[tool.poetry.dev-dependencies]
+alfalfa-client = "0.1.0.dev7"
+#alfalfa-client = {git = "https://git@github.com/nrel/alfalfa-client.git", rev = "develop"}
```

Comment:
@nllong can you delete this?

Reply:
delete the git line? I'd like to keep it that way I don't have to look up the syntax the next time I want to test a new alfalfa-client release.

Reply:
got it ok yeah cool schmart schamart